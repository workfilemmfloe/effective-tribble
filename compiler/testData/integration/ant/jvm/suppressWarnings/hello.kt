package hello

fun foo(p: Int??) {

}

trait T {
    abstract fun foo()
}

fun main(args : Array<String>) {
    println("Hi!")
}
