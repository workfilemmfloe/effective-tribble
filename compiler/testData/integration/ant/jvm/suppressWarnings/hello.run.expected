OUT:
Hi!

Return code: 0
